src/scripts.js

```diff
@@ -52,7 +76,6 @@ $('.filter-btn').click(findCheckedBoxes);
 $('main').click(selectCard);
 $(".my-pantry-btn").click(toggleMenu);
 $(".saved-recipes-btn").click(showSavedRecipes);
-$(".search-btn").click(searchRecipes);
```

Comment:
Should we remove the search button completeley and just make it an image or something?

Reply:
We can do that along with making the filter options hidden as well. Once filter options are hidden there can be 4 cards in each row